application-mocca-calendar-ui/src/main/resources/MoccaCalendar/Code/Macro.xml

```diff
@@ -1093,6 +1161,27 @@ div.calendar-buttons span.buttonwrapper {
 }
 .skin-flamingo .gadget .fc h2 {
   font-size: 140%;
+}
+
+.import-form {
+  padding: 2rem;
+}
+
+.import-form label {
+  margin-bottom: 8px;
```

Comment:
Using px is not ok, it can easily just look ok on your screen but not for others